webextensions/api/management.json

```
@@ -288,10 +288,12 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": false
+                "version_added": "55",
+                "notes": "Before version 56, only extensions whose 'type' is 'theme' are supported."
```

style: If I understand correctly, this refers to an `ExtensionInfo` property. If so, I'd probably mark up the note (and its siblings/cousins below) like this:

```suggestion
                "notes": "Before version 56, only extensions whose <code>type</code> is <code>'theme'</code> are supported."
```

---

Totally makes sense! Thanks for the suggestion!